src/AutoRest.CSharp/Common/Input/Source/SourceInputModel.cs

```diff
@@ -127,5 +137,40 @@ private bool TryGetName(ISymbol symbol, [NotNullWhen(true)] out string? name)
 
             return name != null;
         }
+
+        private async Task<CSharpCompilation?> LoadBaselineContract()
```

Comment:
since the type that is accepting the return value of this method has the type of `Compilation`, we do not really need to make the return type `CSharpCompilation` 
(ref https://github.com/Azure/autorest.csharp/pull/4026/files#r1423306866)